.circleci/config.yml

```diff
@@ -383,9 +383,18 @@ jobs:
       - attach_workspace:
           at: ~/
       - run:
-          name: Generate HTML output for documentation
+          name: Install dependencies
           command: |
-            antora generate site.yml
+            apk --no-cache add git openssh-client
```

Comment:
Would be nice to have these in the Docker image rather than having to install here

Reply:
yeah, agree we need the same thing in several places, it seems like we need to adopt the maintenance of an Antora build image or include Antora in our build image and reuse this one.

Reply:
Added a derived image with pre-installed dependencies so we don't make the build unstable with maybe nonworking installs all time.